Tools/TestWebKitAPI/Tests/WebKitGtk/TestInspector.cpp

```diff
@@ -319,6 +319,7 @@ class CustomInspectorTest: public InspectorTest {
         g_assert_true(GTK_IS_PANED(pane));
         gtk_container_remove(GTK_CONTAINER(pane), GTK_WIDGET(inspectorView.get()));
 #endif
+        resizeView(gMinimumAttachedInspectorWidth, (gMinimumAttachedInspectorHeight + 1) * 4 / 3);
```

Comment:
Why `+1 * 4 / 3`?

Reply:
That's the size we set before to be able to attach the inspector, it's the minimum size the view needs to be able to attach the inspector